src/jit/rationalize.cpp

```diff
@@ -157,19 +157,20 @@ void Compiler::fgFixupArgTabEntryPtr(GenTreePtr parentCall, GenTreePtr oldArg, G
     }
 }
 
-// Rewrite InitBlk involving SIMD vector into stlcl.var of a SIMD type.
+// Rewrite a non-TYP_STRUCT indirection as GT_IND(GT_LEA(obj.op1)), or as a simple
+// lclVar if possible.
 //
```

Comment:
Actually, I see that this is something that should be going away eventually, so I'll just make it clear that this is only for SIMD.

---
In reply to: [77282570](https://github.com/dotnet/coreclr/pull/7027#discussion_r77282570) [](ancestors = 77282570,77282496)